index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <title> 5.4 Contact list </title>
     <link rel="stylesheet" href="styles/main.css" media="screen">
+    <link rel="stylesheet" href="styles/contactinfo.css" media="screen">
```

Comment:
I see _why_ you did this! It is a great idea! As we just talked about today, it will become extremely important to separate these files when we have access to *css preprocessors*, but until then, we do not need to separate them.